ui/src/app/components/runs/runs.component.html

```diff
@@ -57,7 +57,7 @@
         </app-number-range-filter>
       </clr-dg-filter>
     </clr-dg-column>
-    <clr-dg-column [clrDgField]="dagRunColumns.STARTED">
+    <clr-dg-column [clrDgField]="dagRunColumns.STARTED" [clrDgField]="'name'" [clrDgSortOrder]="descSort">
```

Comment:
` [clrDgField]="'name'" ` is there a reason why you have it there ? 
It has been already defined `[clrDgField]="dagRunColumns.STARTED"`

Reply:
no reason at all. fixed